src/flynt/cli.py

```diff
@@ -96,11 +96,25 @@ def main():
     )
 
     parser.add_argument(
-        "src", action="store", nargs="+", help="source file(s) or directory"
+        "src", action="store", nargs="*", help="source file(s) or directory"
     )
 
+    parser.add_argument(
+        "--version",
+        action="store_true",
+        default=False,
+        help="Print the current version number and exit."
+    )
     args = parser.parse_args()
 
+    if args.version:
+        print(__version__)
+        sys.exit(0)
+    elif not args.src:
+        print("flynt: error: the following arguments are required: src")
```

Comment:
This doesn't show `usage: flynt [-h] [-v | -q] [--no-multiline | -ll LINE_LENGTH] [-d] [-tc] [-f] [-a] [-e EXCLUDE [EXCLUDE ...]] src [src ...]` like the original behavior did. (functionality to do this is `parser.print_usage()`).

Reply:
much better, thanks for catching this !